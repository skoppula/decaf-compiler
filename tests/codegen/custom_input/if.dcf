callout printf;
void main ( ) {
  int i, sum;
  int j;
  sum = 0;
  
  if ( sum != 0 ) {
    printf ( "ERROR: for loop is bad (1)\n" );
  }
  printf ( "if OK if no previous output\n" );
}

