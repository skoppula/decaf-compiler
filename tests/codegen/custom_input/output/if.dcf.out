if OK if no previous output
